package Day2_Java.exception;

public class Testdivision {
	public static void main(String[] args) throws InterruptedException,ClassNotFoundException{
		Division obj=new Division();
		Division.division(20, 0);
		obj.justfun();
	}
}
